Source/DafnyCore/Backends/JavaScript/JavaScriptCodeGenerator.cs

```diff
@@ -974,7 +974,7 @@ protected class ClassWriter : IClassWriter {
         return $"{DafnyMultiSetClass}.Empty";
       } else if (xType is SeqType seq) {
         if (seq.Arg.IsCharType) {
-          return "''";
+          return "_dafny.Seq.UnicodeFromString('')";
```

Comment:
Does this still support `--unicode-char=false`?

Reply:
Good call. The answer happens to be "yes", and I really mean "happens to". I changed the code to treat the two cases separately.
In writing more test cases, I also ran across the known non-uniformity with the printing of empty strings in Python. I was able to remove this non-uniformity for the default string.